File: ProgramasPython/secao06/exercicio04.py
alt = float(input("Informe a sua altura: "))
sexo= (input("Informe o sexo (f/m):"))

if sexo.lower() == 'm':
    peso_ideal= (72.7 * alt) - 58
elif sexo.lower() == 'f':
    peso_ideal= (32.1 * alt) - 44.7
else:
    peso_ideal= 0
    print("Sexo não reconhecido.")    
    
print("Seu peso ideal é: {0:.2f}".format(peso_ideal))        

File: ProgramasJava/src/br/com/evertonsantos/secao06/Exercicio04.java
package br.com.evertonsantos.secao06;

import java.util.Scanner;

public class Exercicio04 {

	public static void main(String[] args) {
		float alt, peso_ideal;
		char sexo;
		Scanner teclado= new Scanner(System.in);
		
		System.out.println("Informe sua altura: ");
		alt = teclado.nextFloat();
		
		System.out.println("Informe o sexo m/f: ");
		sexo = teclado.next().charAt(0);
		
		if(sexo =='m') {
			peso_ideal = (float)((alt * 72.7) - 58);
			System.out.printf("Seu peso ideal é %.2f", peso_ideal);
		}
		if(sexo =='f') {
			peso_ideal = (float)((alt * 62.1) - 44.7);
			System.out.printf("Seu peso ideal é %.2f", peso_ideal);
		}
		if((sexo != 'f') && (sexo != 'm')) {
			System.out.println("Informe o sexo corretamente(m/f)...\n");
			peso_ideal = 0;
		}
		teclado.close();
	}
	}
